### PR TITLE
gnupg21 2.1.10

### PR DESCRIPTION
**Pull request by jonnybarnes**
This installs on my El Capitan system.

However, when compiling the built in dirmngr component it was producing an error relating to `dns-stuff`. Adding `adns` as a dependency fixes this.

### CODE REVIEW

**Comment by jonnybarnes** [comment]
According to the Jenkins tests this has the same Mavericks issue as before regarding sqlite. Not sure how to fix that.

Maybe for Mavericks pass the configure argument `--disable-sqlite`?

File: gnupg21.rb [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.1.9.tar.bz2"
-  mirror "https://www.mirrorservice.org/sites/ftp.gnupg.org/gcrypt/gnupg/gnupg-2.1.9.tar.bz2"
-  sha256 "1cb7633a57190beb66f9249cb7446603229b273d4d89331b75c652fa4a29f7b6"
-
-  bottle do
-    revision 1
-    sha256 "3c320e58e0ac25ce4536c4c0488e0293b89bf39a38b7068a0099bd51d99ac8f2" => :el_capitan
-    sha256 "17eb793b2e9f488177f7317f09fddb71e613db75d4070479ccb440f2875f2c53" => :yosemite
-    sha256 "eed4a8b01647835d2c07bcaa0a9664ab12b69e0b6718a63db32b2084217b0251" => :mavericks
-  end
```
Comment (mikemcquaid) [inline gnupg21.rb]: Please leave the bottle-block as-is; it'll be updated automatically when pulled.

**Comment by mikemcquaid** [comment]
> Maybe for Mavericks pass the configure argument --disable-sqlite?

Or maybe add a conditional `sqlite` dependency there?

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/a6427571de411d3444904d01352efd199d3bf1e2. Thank you for your contribution to Homebrew @jonnybarnes; we appreciate it! :smiley_cat:

**Pull request closed without merging**